src/transferwise_clj/api/transferwise/core.clj

```diff
@@ -0,0 +1,17 @@
+(ns transferwise-clj.api.transferwise.core
+  (:require [clj-http.client :as client]
+            [transferwise-clj.data.core :as data]))
+
+(defn post-quote
+  "Retrieve the quote object from Transferwise"
+  [url quote]
+  (client/post url {:form-params  quote
+                    :content-type :json
+                    :as           :json}))
+
+(defn parse-quote
+  "Parse transferwise post response into new-rate-map"
+  [{:keys [body]}]
+  (let [{rate    :rate
+         options :paymentOptions} body]
```

Comment:
This would be more trouble than it's worth but I'll leave it here anyways: I'd rather have one formatting standard (kebab-case) and transform everything at the borders than have to guess which format each map uses.

Reply:
I don't think that I will implement that right now. Once this is only used for Transferwise transactions, so for me, it's ok to keep as it is. But totally got your point :)